spec/features/comment_pages/comment_feed_spec.rb

```diff
@@ -48,18 +58,24 @@
 
     it 'allows adding a comment' do
       submit_form
+
+      within comment_feed do
+        expect(Comment.last.content).to eq 'test comment'
+      end
     end
 
-    describe 'local caching', js: true do
-      let!(:original_page_with_comment_path) { page.current_path }
+    include_examples 'creates an Activity', :create, Comment
 
+    describe 'local caching' do
```

Comment:
I think we can improve this spec by consolidating the two `it` blocks. In one `it block, first fill in the data then refresh. Assert that the data is cached. Then, submit the data then refresh. Assert that the cache is cleared.

Reply:
Btw, there's already a `local caching` describe block somewhere in the spec.

Reply:
Gotcha